promscale/installation/kubernetes.md

```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
 `helm install`.
 </highlight>
 
+By default, the `timescaledb-single` helm chart deploys TimescaleDB in 
+high availability mode. By default, this creates three database replicas, 
+which consumes three times the amount of disk space. Each database 
+instance mounts to its own persistent volume claim (PVC). 
+
+You can 
+disable high availability mode by changing the `timescaledb-single` 
+helm chart `replicaCount` values to `1` in [`values.yaml`][timescaledb-single-values-yaml].
+
+<procedure>
+
+#### Steps to disbale TimescaleDB High-Availability
+1.  Download the default `values.yaml` of `timescaledb-single` helm chart from [here][timescaledb-single-values-yaml].
```

Comment:
```suggestion
1.  Download the default `values.yaml` for the `timescaledb-single` Helm chart. To get the
file, see the [`timescaledb-kubernetes` repository][timescaledb-single-values-yaml].
```

Reply:
I updated this some @charislam. We aren't actually directing them to the repository so the second sentence didn't make sense and this is just direct. "download the file..."